modules/appeals_api/app/services/appeals_api/pdf_construction/notice_of_disagreement/pages/hearing_type_and_additional_issues.rb

```diff
@@ -43,12 +44,22 @@ def extra_issues?
             form_data.contestable_issues.count > MAX_ISSUES_ON_FIRST_PAGE
           end
 
+          def short_preferred_email?
+            form_data.preferred_email.length <= 120
```

Comment:
Can someone remind me why we are referring to this as `preferred` email as opposed to `veteran_email` or `email`?

Reply:
The name in the pdf file? Or we named the field? Just wondering.

Reply:
Yes, the "physical" 10182 pdf file uses the term `My Preferred E-Mail Address` for the field.